tsl/src/compression/create.c

```diff
@@ -1063,10 +1063,19 @@ tsl_process_compress_table_add_column(Hypertable *ht, ColumnDef *orig_def)
 	Oid coloid;
 	int32 orig_htid = ht->fd.id;
 	char *colname = orig_def->colname;
-	TypeName *orig_typname = orig_def->typeName;
 
+	FormData_hypertable_compression *ht_comp =
+		ts_hypertable_compression_get_by_pkey(orig_htid, colname);
+	/* don't add column if it already exists */
+	if (ht_comp)
+	{
+		return;
+	}
```

Comment:
```suggestion
	if (ht_comp)
		return;
```
I'm pretty sure the format check will pass if you simplify it this way.